src/work-stacking.lisp

```diff
@@ -0,0 +1,58 @@
+;;;; Scratch work for 'better' stacking functions.
```

Comment:
Can this file be deleted from the repo?

Reply:
Oops.

Oh yeah, that'll be gone before I convert this from a Draft PR to a normal PR.